samples/drivers/kscan/src/main.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <gpio.h>
+#include <soc.h>
+#include <kscan.h>
+
+#define LOG_LEVEL LOG_LEVEL_DBG
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(main);
```

Comment:
better to use `LOG_MODULE_REGISTER(main, LOG_LEVEL_DBG);`
then `#define LOG_LEVEL LOG_LEVEL_DBG` can be removed.